bootstrap/api/requests.go

```diff
@@ -0,0 +1,116 @@
+//
+// Copyright (c) 2018
+// Mainflux
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package api
+
+import "github.com/mainflux/mainflux/bootstrap"
+
+type apiReq interface {
+	validate() error
+}
+
+type addReq struct {
+	key         string
+	ExternalID  string   `json:"external_id"`
+	ExternalKey string   `json:"external_key"`
+	Channels    []string `json:"mainflux_channels"`
```

Comment:
Remove `mainflux` prefix, I think that you don't need it (correct me if I'm wrong).